### PR TITLE
Add website- and storeview-code in stores admin grid

### PR DESCRIPTION
**Pull request by aschrammel**
This PR adds the webiste- and storeview-code in the stores admin grid like in Magento 1 stores overview.

### Description
The PR joins additionally the `code` field from the `store` table to the admin grids row and adds the website code as well as the store view code in the respective renderers.

### Fixed Issues (if relevant)
-

### Manual testing scenarios
1. Go to "Stores" > "Settings / All Stores"
2. Before applying the PR there are only listed Website, Store and StoreView by their names
3. After applying the PR there are the respective codes shown beneath the Website and StoreView names

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by mzeis** [comment]
Hi @aschrammel,

thank you very much for your contribution! Would you consider adding the store name, too?

**Comment by aschrammel** [comment]
Hi @mzeis,

thanks for your feedback. 
I think you mean the store code, as the name is already shown - am I right?

**Comment by mzeis** [comment]
@aschrammel Exactly :)

**Comment by aschrammel** [comment]
@mzeis Done :)

**Comment by mzeis** [comment]
@magento-engcom-team give me new test instance

**Comment by magento-engcom-team** [comment]
Hi @mzeis, here is your [new test Magento instance](http://pr-14156.engcom.dev.magento.com)

File: app/code/Magento/Backend/Block/System/Store/Grid/Render/Group.php [hunk 1]
```diff
@@ -27,6 +27,7 @@ public function render(\Magento\Framework\DataObject $row)
         $this->getUrl('adminhtml/*/editGroup', ['group_id' => $row->getGroupId()]) .
         '">' .
         $this->escapeHtml($row->getData($this->getColumn()->getIndex())) .
-        '</a>';
+        '</a><br />'
+        . '(Code: ' . $row->getGroupCode() . ')';
```
Comment (mzeis) [inline app/code/Magento/Backend/Block/System/Store/Grid/Render/Group.php]: @aschrammel Please can you make "Code" translatable in all 3 places (like "Edit Store" here: [Group.php](https://github.com/magento/magento2/blob/2.2-develop/app/code/Magento/Backend/Block/System/Store/Grid/Render/Group.php#L23-L25) and [en_US.csv](https://github.com/magento/magento2/blob/2.2-develop/app/code/Magento/Backend/i18n/en_US.csv)?

Then we should be good to go. Thanks!

**Review by mzeis — CHANGES_REQUESTED** [review]

**Comment by aschrammel** [comment]
@mzeis Code is translatable now 🌐

The translation in [en_US.csv](https://github.com/magento/magento2/blob/2.2-develop/app/code/Magento/Backend/i18n/en_US.csv#L101) is already given on line 101.

**Comment by mzeis** [comment]
@magento-engcom-team give me new test instance

**Comment by magento-engcom-team** [comment]
Hi @mzeis, here is your [new test Magento instance](http://pr-14156.engcom.dev.magento.com)

**Review by mzeis — APPROVED** [review]

**Comment by mzeis** [comment]
@aschrammel Thanks for you improvements, looking good! 👍 I'm handing over to QA & merging.

**Comment by magento-engcom-team** [comment]
Hi @mzeis, thank you for the review.
[ENGCOM-968](https://jira.corp.magento.com/browse/ENGCOM-968) has been created to process this Pull Request

**Comment by aschrammel** [comment]
@magento-engcom-team thanks for acceptance 😄
In #14157 is the respective forward port to 2.3 if wanted.

**Comment by mzeis** [comment]
Thanks! I'll process it once the merge for this one is completed :)

**Comment by mzeis** [comment]
Hi @aschrammel, thanks again for your PR! It has been merged into `2.2-develop`. We now can proceed with #14157.

**Pull request merged**